migrations/2017_11_26_015000_create_user_roles_table.php

```diff
@@ -13,12 +13,16 @@ class CreateUserRolesTable extends Migration
      */
     public function up()
     {
+        Schema::table('users', function (Blueprint $table) {
```

Comment:
We shouldn't be modifying the `users` table in a `create_user_roles_table` migration.  That will just confuse people

Also, we can't edit pre-existing migrations as upgrading users won't be able to run it.